frontend/src/components/drawerfield/Card.tsx

```diff
@@ -1,8 +1,14 @@
-import { CardTransform } from '../data/CardTransform';
+import { CardTransform } from '../../data/CardTransform';
 import { MouseEventHandler, CSSProperties } from 'react';
 
-export default function Card({ transform, selectCallback }: { transform: CardTransform, selectCallback: MouseEventHandler }) {
-    const image = require('../assets/card.png');
+export default function Card({
+    transform,
+    selectCallback,
+}: {
+    transform: CardTransform;
+    selectCallback: MouseEventHandler;
+}) {
+    const image = require('../../assets/card.png');
```

Comment:
Ha a függvény-komponensen belül van az import, akkor minden renderelésnél újra be kell töltenie.
Ezt vidd ki a többi import közé.
Esetleg használd az új import stílust:
```ts
import image from '../assets/card.png'
```

Reply:
[b369aa0](https://github.com/szte-agilis/imagine/pull/6/commits/b369aa0a3b262b0dfd141d85220bb069f5646779)